plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeTransformUtils2.kt

```diff
@@ -0,0 +1,14 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Do we want to change this file name or include it in the already available utils file?

Reply:
if you keep the same name then you get a runtime error because there are 2 classes with the same name